tests/Core/Tokenizer/BackfillFnTokenTest.inc

```diff
@@ -73,6 +73,49 @@ fn(array $a) : array => $a;
 /* testTernary */
 $fn = fn($a) => $a ? /* testTernaryThen */ fn() : string => 'a' : /* testTernaryElse */ fn() : string => 'b';
 
+/* testConstantDeclaration */
```

Comment:
This new test was added but is not checked anywhere. Should it be added to dataNotAnArrowFunction()?

Reply:
Resolved via #2863